### PR TITLE
FEATURE: display Latest date of validated data

### LINKED ISSUE #38: display Latest date of validated data
The date of latest validated data is equal to the date of latest "Tribal Fusion" (partenaire) record. Then we should retrieve this date on db and display beside range dates on report page:

"`Complete compiled data up to`" [DATE]

And the translation on dictionary (`/src/assets/i18n/fr.json`) "`Données compilées complètes jusqu'au`"

### PR DESCRIPTION
**Pull request by danconte72**
FEATURE: Closes #38, display Latest date of validated data
Changes:

- Created a HTTP Request in CloudFunctions to update the latest updated Date;
- Added getLatestUnpluggedImport function to get the proper value on firebase and set in sites-report;

### CODE REVIEW

File: ionic.config.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "name": "ion-ams-report",
   "app_id": "0451e327",
-  "type": "ionic-angular"
+  "type": "ionic-angular",
+  "integrations": {
```
Comment (vdias38) [inline ionic.config.json:5]: What's the purpose of this field (integrations) ?

File: firebase/functions/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const functions = require('firebase-functions');
+const sha1 = require('sha1');
+const format = require("date-format-lite")
```
Comment (vdias38) [inline firebase/functions/index.js:3]: do you need this package ?

**Review by vdias38 — COMMENTED** [review]
this commit should not close the #38 issue, It's a feature commit necessary should use a title like "FEATURE: add http endpoint to save today date on db, settings/latestUnpluggedImport"

What's the url to call ?

File: firebase/functions/index.js (hunk 2 above)
Comment (danconte72) [reply firebase/functions/index.js:3]: - firebase-functions to use cloud-functions
- sha1 - to encrypt the token
- format - to format the date

File: ionic.config.json (hunk 1 above)
Comment (danconte72) [reply ionic.config.json:5]: integrations was generated automatically by ionic serve cmd

**Comment by danconte72** [comment]
The url will be generate by the deploy event.

**Pull request merged**